cli/loader/src/lib.rs

```diff
@@ -18,10 +19,42 @@ use tree_sitter_tags::{Error as TagsError, TagsConfiguration};
 #[derive(Default, Deserialize, Serialize)]
 pub struct Config {
     #[serde(default)]
-    #[serde(rename = "parser-directories")]
+    #[serde(
+        rename = "parser-directories",
+        deserialize_with = "deserialize_parser_directories"
+    )]
     pub parser_directories: Vec<PathBuf>,
 }
 
+// Replace `~` or `$HOME` with home path string.
+// (While paths like "~/.tree-sitter/config.json" can be deserialized,
+// they're not valid path for I/O modules.)
+fn deserialize_parser_directories<'de, D>(deserializer: D) -> Result<Vec<PathBuf>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let paths = Vec::<PathBuf>::deserialize(deserializer)?;
+    let home = match dirs::home_dir() {
+        Some(home) => home,
+        None => return Ok(paths),
+    };
+    let standarized = paths
+        .into_iter()
+        .map(|path| standarize_path(path, &home))
+        .collect();
```

Comment:
One last nit, I promise! :smile:  `standardized` and `standardize_path` are misspelled. (missing `d`)

Otherwise this looks great, thanks!

Reply:
Thank you :) Which would you think is better, standardize or normalize?

Reply:
Thank you for the review! I fixed the misspellings and force pushed.

Reply:
No strong opinion between standardize and normalize, happy to go with what you wrote.  Thanks for the patch!

Reply:
Then I'd go with standardize. Thank you <3